static/app/views/issueDetails/groupReplays/groupReplays.spec.tsx

```diff
@@ -597,22 +597,19 @@ describe('GroupReplays', () => {
         );
       });
 
-      // browserHistory.replace = jest.fn();
-
       const mockReplace = jest.mocked(browserHistory.replace);
-      await waitFor(() => {
-        const replayPlayPlause = screen.getAllByTestId('replay-table-play-button')[0];
-        return act(() => userEvent.click(replayPlayPlause));
-      });
-
-      await tick();
-      expect(mockReplace).toHaveBeenCalledWith(
-        expect.objectContaining({
-          pathname: '/organizations/org-slug/replays/',
-          query: {
-            selected_replay_index: 1,
-          },
-        })
+      const replayPlayPlause = screen.getAllByTestId('replay-table-play-button')[0];
+      await act(() => userEvent.click(replayPlayPlause));
```

Comment:
userEvent calls act for you, which errors in react 18 with overlapping acts. if its required there's likely another state change not being waited for before the click
```suggestion
      const replayPlayPlause = screen.getAllByTestId('replay-table-play-button')[0];
      await userEvent.click(replayPlayPlause);
```